src/main/java/edu/hawaii/its/api/groupings/GroupingGroupOwners.java

```diff
@@ -0,0 +1,55 @@
+package edu.hawaii.its.api.groupings;
+
+import edu.hawaii.its.api.wrapper.GetMembersResult;
+import edu.hawaii.its.api.wrapper.SubjectsResults;
+
+/**
+ * When getMembers is called, GroupingGroupOwners holds the information about UH affiliates that are listed in a
+ * group such as include, exclude, owners.
+ */
+public class GroupingGroupOwners implements GroupingResult {
```

Comment:
I think it makes more sense to call the class "GroupingOwnerMembers" based on the naming of the other class, GroupingGroupMembers. This class you made is specific to the owner group, so "group" should be replaced with "owner" rather than replacing "members".